haskell-xmpp.cabal

```diff
@@ -1,17 +1,17 @@
 Name: haskell-xmpp
-Version: 1.1.0
+Version: 2.0.0
 License: BSD3
 License-File: LICENSE
-Author: Dmitry Astapov <dastapov@gmail.com>, pierre <k.pierre.k@gmail.com>
-Maintainer: Dmitry Astapov <dastapov@gmail.com>
-Homepage: http://patch-tag.com/r/adept/haskell-xmpp/home
+Author: Dmitry Astapov <dastapov@gmail.com>, pierre <k.pierre.k@gmail.com>, Jappie Klooster <jappie@riskbook.com>
+Maintainer: Jappie Klooster <jappie@riskbook.com>, Veronika Romashkina <veronika@riskbook.com>
+Homepage: https://github.com/riskbook/haskell-xmpp
 Category: Network
 Copyright: (c) 2005-2011 Dmitry Astapov, k.pierre
 Stability: Stable
-Cabal-version: >=1.6
+Cabal-version: >=1.10
```

Comment:
Let's use at least 2.0, migrate to the newer syntax and put this at the first line 🙂

Reply:
I just did this because hackage complained. I'm not sure what it means.

Reply:
It the version of `Cabal`, it is really an ancient one. Let's use newer and also put it as the first item in the cabal file as it is really important.

Reply:
Is 2 now